src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -15,6 +15,7 @@ class Kilosort4Sorter(BaseSorter):
 
     sorter_name: str = "kilosort4"
     requires_locations = True
+    gpu_compatibility = "nvidia-required"
```

Comment:
In theory nvidia is not require because torch also with cpu which slower but good for for testing on laptop wihtout nvidia for instance. So maybe we should add "nvidia-recomended" case that propagate the nvidia docker option only if nvidia is present.